src/Panel/Ui/Stat.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Kirby\Panel\Ui;
+
+use Kirby\Cms\ModelWithContent;
+use Kirby\Exception\InvalidArgumentException;
+use Kirby\Toolkit\I18n;
+
+/**
+ * @package   Kirby Panel
+ * @author    Bastian Allgeier <bastian@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ * @since     5.1.0
+ */
+class Stat extends Component
+{
+	public function __construct(
+		public ModelWithContent $model,
+		public array|string $label,
+		public array|string $value,
+		public string $component = 'k-stat',
+		public string|null $icon = null,
+		public array|string|null $info = null,
+		public array|string|null $link = null,
+		public string|null $theme = null,
+	) {
+	}
+
+	public static function from(
```

Comment:
`Nitpick:` Wondering if `factory` would be more fitting, but no strong opinion here.

Reply:
Ignore it. We have `from` and `for` also in many places.